README.MD

```diff
@@ -1,57 +1,58 @@
 # tsoa
+
 Pronounced so·uh
 
 [![Codeship Status for lukeautry/tsoa](https://codeship.com/projects/cdce38d0-1f6b-0134-258e-1ed679ae6c9d/status?branch=master)](https://codeship.com/projects/160322)
 [![npm version](https://badge.fury.io/js/tsoa.svg)](https://badge.fury.io/js/tsoa)
 
 ## Table of Contents
 
-  * [Goal](#goal)
```

Comment:
It seems like you have a formatter running on save, would you mind resubmitting this PR without the noise?

Reply:
@WoH I'll clear the formatting and resubmit.